src/main/java/task/TaskSchedulerRunner.java

```diff
@@ -6,15 +6,29 @@
 import task.core.TaskManager;
 
 /**
- * Created by ahmetkucuk on 3/10/18.
+ * Class that attempts to connect to Default DB until connection is successful. Task
+ * manager is started and sends tasks to HEK puller and image puller and creates
+ * a JobRecord object in Monitoring DB every 30 seconds. Monitoring App is
+ * started, which creates an HTTP server on port 1278. The app provides a list
+ * of JobRecords created in the DB.
+ * @author - ahmetkucuk
+ * @author - kqian5
  */
 public class TaskSchedulerRunner {
 
-    public static void main(String[] args) {
-        System.out.println("Task Scheduler Runner Started!");
-        DBPrefs.waitDefaultDBConnections();
-        TaskManager.instance().startWithFixedRate();
-        JobRecordConnectionProvider.connection().createJobRecordTable();
-        MonitoringRunner.startServer();
-    }
-}
+	/**
```

Comment:
I think you should move lines 19 - 33 in an indent space. And the function is missing a closing bracket?

Reply:
I was wrong about there being no closing bracket haha sorry!

Reply:
wait what do you mean by move in an indent space? I tried to align the comments with the beginning of the method header line.